src/test/ui/rfcs/rfc-2396-target_feature-safe-calls.rs

```diff
@@ -0,0 +1,19 @@
+#[target_feature(enable = "sse2")]
+fn foo() {}
+
+#[target_feature(enable = "sse2")]
```

Comment:
And this too, let's exercise it for inherent methods (including in the run-pass test).